### ISSUE TITLE
Netwatcher ID should be node address

### ISSUE THREAD

**Issue by jeandemeusy** [hoprnet/ct-research#193]
### Syntax
- `peer` refers to community node "in-the-wild"
- `node` referes to a HOPR node that are used to access the existing network

### Description of the issue
The `Netwatcher`s gather metrics about peers they are able reach. This gathering forms a dictionary that is unique to each `Netwatcher`. At some point, this list is sent to the `Aggregator` along with a `nw-id`, and based on a custom load balancing algorithm, sorts, for each **peer** the `Netwatcher`s which see it. This list is used later on, when coming to distributing messages.

Distributing messages is done by a separate "module", called a `Postman`. For each node already used with a `Netwatcher`, a `Postman` is also attached. One can see this as having two sidecars attached to each ct-App nodes. The `Postman` task has a dedicated name related to the node that needs to send message to a specific peer, so that with the information the `Netwatcher` sends to the `Aggregator`, the right `Postman` is being reached out for each task created by the `EconomicHandler`

What happens if the `Netwatcher` goes down (not his associated node) while the app is running ? In the database's table, the dead-`Netwatcher`'s id will be still stored. If a new instance of this `Netwatcher` is being instantiated, it will have a brand new id (random string), that will be different than the one before. This could lead to a mess in the db. Even more, a task for the old `Netwatcher` instance will never be picked-up by the new one.

### Solution to implement 
**TL;DR: the unique identifier that needs to be sent to the db by the `Netwatcher` is simply the connected node address.**  
In the case of a `Netwatcher` going down, as the stored information is the node address, when the new instance will be running for the same node, the identifier will be linked to the new instance instead. 

This also reduce complexity of the `Postman`: for now, its task is defined using a name linked to the `Netwatcher` that have seen the peer. But actually, the `Postman` does not care about which `Netwatcher` has seen a peer, only about which node has seen the peer.
